prisma/schema.prisma

```diff
@@ -0,0 +1,26 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider  = "postgresql"
+  url       = env("DATABASE_URL")
+  directUrl = env("DIRECT_URL")
+}
+
+model User {
+  id        String   @id @default(uuid())
+  createdAt DateTime @default(now()) @map("created_at")
+  name      String
+  email     String   @unique
+  password  String
+  updatedAt DateTime @updatedAt @map("updated_at")
+}
+
+model UserAuthProvider {
+  id           String   @id @default(uuid())
+  userId       String   @map("user_id")
```

Comment:
I think, we didn't agree `@map` usage.
This is a cool feature, I like it, just lets make an agreement on how we will use mapping.